feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,50 @@
+<!--
+  ~ Copyright 2019 Red Hat, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<!--
+If you want to use a Galleon 'universe' (see the Galleon documentation) you could express that there instead.
+-->
+<build xmlns="urn:wildfly:feature-pack-build:3.1" producer="org.wildfly.extras.elytron-tls:elytron-tls-feature-pack">
+    <transitive>
+        <dependency group-id="org.wildfly" artifact-id="wildfly-ee-galleon-pack">
```

Comment:
If we switch to `wildfly-preview-feature-pack`, updates would be needed here too (e.g., see https://github.com/wildfly-extras/wildfly-grpc-feature-pack/blob/main/feature-pack/wildfly-feature-pack-build.xml).